.travis.yml

```diff
@@ -9,11 +9,21 @@ matrix:
     - os: osx # OSX 10.11
       osx_image: xcode7.2
       dotnet: 1.0.0-preview2-003121
+    - os: osx # OSX 10.12
+      osx_image: xcode7.3
+      dotnet: 1.0.0-rc4-0044771
```

Comment:
Please CMIIW, but I think it's `0044771` -> `004771`.

It will work after travis-ci/travis-build#973 :)